cloud/pkg/cloudstream/tunnelserver.go

```diff
@@ -17,37 +17,51 @@ limitations under the License.
 package cloudstream
 
 import (
+	"context"
 	"crypto/tls"
 	"crypto/x509"
 	"encoding/pem"
 	"fmt"
 	"net/http"
+	"os"
 	"strings"
 	"sync"
 	"time"
 
 	"github.com/emicklei/go-restful"
 	"github.com/gorilla/websocket"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
 	certutil "k8s.io/client-go/util/cert"
 	"k8s.io/klog/v2"
 
 	hubconfig "github.com/kubeedge/kubeedge/cloud/pkg/cloudhub/config"
 	streamconfig "github.com/kubeedge/kubeedge/cloud/pkg/cloudstream/config"
+	"github.com/kubeedge/kubeedge/cloud/pkg/common/client"
 	"github.com/kubeedge/kubeedge/pkg/stream"
 )
 
+const (
+	// The amount of time the tunnelserver should sleep between retrying node status updates
+	retrySleepTime        = 20 * time.Millisecond
```

Comment:
```suggestion
	retrySleepTime        = 20 * time.Second
```
The retry time can be 20s here, if there is a delay in edge node creation, here need to wait more.

Reply:
Done